src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,268 @@
+package duke;
+
+import java.io.*;
```

Comment:
Try to import only the packages you are using instead of the whole java.io package